attribute/image.go

```diff
@@ -30,19 +30,22 @@ var mimeTypeMap = map[ImageType]string{
 
 // GetMIMEType returns the MIME type of this piece of Yoti user information. For more information see:
 // https://en.wikipedia.org/wiki/Media_type
-func GetMIMEType(imageType ImageType) (result string) {
+func GetMIMEType(imageType ImageType) (string, error) {
 	if val, ok := mimeTypeMap[imageType]; ok {
-		return val
+		return val, nil
 	}
 
-	log.Printf("Unable to find a matching MIME type for value type %q", imageType)
-	return
+	return "", fmt.Errorf("Unable to find a matching MIME type for value type %q", imageType)
 }
 
 // Base64URL is the Image encoded as a base64 URL
 func (image *Image) Base64URL() (string, error) {
 	base64EncodedImage := base64.StdEncoding.EncodeToString(image.Data)
-	contentType := GetMIMEType(image.Type)
+	contentType, err := GetMIMEType(image.Type)
```

Comment:
Defining our own enum for MIME type doesn't add anything.  We can just keep the type on the Image as a String.

Reply:
Will change to string